toolkits/global/packages/global-corporate-footer/README.md

```diff
@@ -1,29 +1,120 @@
 # Global corporate footer
 
-## Branding
+[![NPM version][badge-npm]][info-npm]
 
-To include `global-corporate-footer` in your application, you need to choose **ONE** brand from those available. The `DEFAULT` brand is included in all other brands, and any settings that are not configured will fall back to default.
+The Global corporate footer component aims to deliver a consistent footer
+experience across our services and therefore aid navigation for our users.
+
+It consists of:
+- a set of standard links
+- our corporate logo (Springer Nature)
+- a copyright line which includes the current year followed by "Springer Nature".
+
+## When to use this component
+
+Use the Global corporate footer component as soon as your product or service
+needs a footer with a set of standardized links.
+
+## Installation
+
+To use the Global corporate footer component, enter the following command in
+your Terminal:
+
+```
+npm install @springernature/global-corporate-footer
+```
+
+Import the Sass:
+
+**Note**: The Global footer does not include settings overrides per brand as
+other components usually do. This is intentional and in order to have it
+consistent across brands.
 
 ```scss
-// Pick ONE of the brands below to include
 @import '@springernature/global-corporate-footer/scss/10-settings/default';
-@import '@springernature/global-corporate-footer/scss/10-settings/springer';
-@import '@springernature/global-corporate-footer/scss/10-settings/nature';
+
+// Include this with your other mixins
+@import '@springernature/brand-context/default/scss/30-mixins/lists';
```

Comment:
You don't need this in here as you will get this via the `brand-context` import that all apps need to have.